cwe_checker_rs/src/intermediate_representation/expression/builder.rs

```diff
@@ -1,8 +1,47 @@
-use super::{BinOpType, Expression};
+use apint::ApInt;
+
+use super::{BinOpType, CastOpType, Expression, Variable};
 use crate::prelude::*;
 
 /// ## Helper functions for building expressions
 impl Expression {
+    // Shortcut for creating a constant expression from an i64 value
+    pub fn const_from_i64(value: i64) -> Expression {
+        Expression::Const(Bitvector::from_i64(value))
+    }
+
+    // Shortcut for creating a constant expression from an apint value (e.g. copy of global address)
+    pub fn const_from_apint(value: ApInt) -> Expression {
+        Expression::Const(value)
+    }
+
+    // Shortcut for creating a variable expression
+    pub fn var(name: &str) -> Expression {
+        Expression::Var(Variable {
+            name: name.into(),
+            size: ByteSize::new(8),
+            is_temp: false,
+        })
+    }
+
+    // Shortcut for creating a cast expression
+    pub fn cast(self, op: CastOpType) -> Expression {
+        Expression::Cast {
+            op,
+            size: ByteSize::new(8),
+            arg: Box::new(self),
+        }
+    }
```

Comment:
We should either add `size` as a parameter to both functions or make them `#[cfg(test)]`, since the implicit size assumption of 8 bytes would introduce very hard to spot errors if used in non-unit-test-code.